plover/dictionary/test_default_dict.py

```diff
@@ -4,7 +4,7 @@
 import json
 import unittest
 
-DICT_PATH='plover/assets/dict.json'
+DICT_PATH='plover/assets/dict_unsorted.json'
```

Comment:
This doesn't really fix the test. 

The default dictionary now consists of all the .json files combined. The test would need to load all of them rather than just dict_unsorted.json.